tee-worker/core-primitives/node-api/metadata/src/lib.rs

```diff
@@ -56,20 +56,21 @@ pub trait NodeMetadataTrait:
 	+ BalancesCallIndexes
 	+ TimestampCallIndexes
 {
+	fn get_runtime_version(&self) -> u32;
 }
 
-impl<
-		T: TeebagCallIndexes
-			+ IMPCallIndexes
-			+ VCMPCallIndexes
-			+ SystemSs58Prefix
-			+ UtilityCallIndexes
-			+ ProxyCallIndexes
-			+ BalancesCallIndexes
-			+ TimestampCallIndexes,
-	> NodeMetadataTrait for T
-{
-}
+// impl<
```

Comment:
Can this be removed?

Reply:
Possibly. @Kailai-Wang suggested having the method in the `SystemSs58Prefix` trait tho (which should then probably be renamed to sth like `SystemPalletTrait`) so I'm having a look at that; if that works then I can keep the old impl

Reply:
Fixed :D